src/index.js

```diff
@@ -32,9 +32,11 @@ function resolveConfig({
         .then(({ config }) => resolve(config, prettierOptions));
 }
 
-resolveConfig.resolve = (stylelintConfig, prettierOptions = {}) => {
```

Comment:
Why'd you get rid of the default parameter? It's a bad practice to mutate the function arguments. 😕

Reply:
Because that's the thing that was breaking it 😅 . The default argument is used only if the argument passed is `undefined`. In this case, it is sometimes `null`. This breaks the function's assumption that `prettierOptions` is an object in all cases. 

`prettierOptions = prettierOptions || {};` makes sure it's an object when both `null` or `undefined`. Reading this again I should've made that more obvious. Maybe:
```javascript
if (prettierOptions === null || typeof prettierOptions !== 'object') {
	prettierOptions = {};
}
```

This project has no pulse so normally I would ignore this ambiguity but I just noticed I'm mutating a function argument which one should _never_ do 🙈 . I'll update the PR.